docs/topics/rn-new-features.adoc

```diff
@@ -7,21 +7,46 @@
 
 This section describes the new features of the {ProductName} ({ProductShortName}) {ProductVersion}.
 
-.MTA plugin for IntelliJ IDEA
+.Rules for migrating from Java EE8 to Jakarta EE9
+{ProductShortName} now includes a comprehensive set of rules to support the Java EE8 to Jakarta EE9 migration path by:
 
-The current release supports the {ProductName} plugin for IntelliJ IDEA. The plugin provides assistance directly in IntelliJ IDEA for developers making changes for a migration or modernization effort.
+* Replacing `javax` artifact dependencies declared within `pom.xml` files
+* Replacing `javax` package import statements
+* Renaming `javax` XML schema namespaces
+* Renaming `javax` prefixed properties
+* Renaming bootstrapping files
 
-The plugin is available through the link:https://plugins.jetbrains.com[IntelliJ marketplace].
+Nearly all these new rules have quick fixes so the changes that they support can be automated when using an {ProductShortName} plugin or extension within an IDE (Red Hat CodeReady Studio, Eclipse, Eclipse Che, IntelliJ or Microsoft Visual Studio Code).
 
-.Rules for migrating from JBoss EAP 7.3 to 7.4
+The rules have two targets: `jakarta-ee:9` and `eap:8.` However, using the target `jakarta-ee` is sufficient for all of them.
```

Comment:
Not sure this belongs in a release note. This is more of a documentation comment.